cache/in-memory/src/event/interaction.rs

```diff
@@ -80,6 +81,8 @@ mod tests {
 
     #[test]
     fn test_interaction_create() {
+        let timestamp = Timestamp::from_secs(1_632_072_645).expect("non zero");
```

Comment:
nit: tests shouldn't mix string and integer parsing

Reply:
it's useful to test both imo